### PR TITLE
[WIP] Messaging status

### PR DESCRIPTION
**Pull request by sanchaymittal**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by wanglonghong — APPROVED** [review]
Looks good to me lol

**Review by rhlsthrm — APPROVED** [review]

**Comment by rhlsthrm** [comment]
Looks good I will add to ops.

File: ops/testnet/staging/backend/main.tf
Comment (rhlsthrm) [inline ops/testnet/staging/backend/main.tf:1]: Need to add this to all environments, testnet prod and mainnet.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: ops/testnet/staging/backend/main.tf
Comment (wanglonghong) [reply ops/testnet/staging/backend/main.tf:1]: https://github.com/connext/monorepo/pull/3683/commits/383cd5c4d669ea4a5118d93f35e62bca57cbd7de

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**